src/components/tabs/js/tabScroll.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+  'use strict';
+  angular.module('material.components.tabs')
+      .directive('mdTabScroll', MdTabScroll);
+
+  function MdTabScroll () {
```

Comment:
The implementation seems weird (creating a new scope every time an event happens etc).
Not ot mention sub-optimal from a performance perspective.

Why not follow the paradigm of the rest of the `ngEvent` directives ?
(Reference: https://github.com/angular/angular.js/blob/master/src/ng/directive/ngEventDirs.js)